core/src/main/kotlin/com/gatehill/corebot/chat/model/template/CustomActionTemplate.kt

```diff
@@ -18,7 +18,7 @@ abstract class CustomActionTemplate : BaseActionTemplate() {
             val options = transform(actionConfig, placeholderValues)
             CustomAction(actionType,
                     buildShortDescription(actionConfig),
-                    if (actionMessageMode == ActionMessageMode.INDIVIDUAL) buildStartMessage(options, actionConfig) else null,
+                    if (actionMessageMode == ActionMessageMode.INDIVIDUAL && actionConfig.showJobOutcome) buildStartMessage(options, actionConfig) else null,
```

Comment:
@outofcoffee @pcornish this has been requested multiple times in my org as simple jobs like enable or disable stuff clutters a lot of output and the info is repeated by the reactions